sydent/http/srvresolver.py

```diff
@@ -108,7 +108,7 @@ def resolve_service(self, service_name):
         :type service_name: bytes
 
         :returns a list of the SRV records, or an empty list if none found.
-        :rtype: Deferred[list[Server]]
+        :rtype: Deferred[[Server]]
```

Comment:
Why? This isn't the right format?

Reply:
Should it be uppercase? It's quite inconsistent across the codebase.

Reply:
So it should be the types, i.e. if `Server` is an object then yes, if `Server` is actually say a string then it should be `Deferred[list[str]]` etc

Reply:
Ah ok, I see. Deferred is capitalized, but `list` is a built-in which is lowercase.